game/screen_paths.cc

```diff
@@ -8,66 +8,151 @@
 #include <boost/thread.hpp>
 #include <boost/bind.hpp>
 
-ScreenPaths::ScreenPaths(std::string const& name, Audio& audio): Screen(name), m_audio(audio) {}
+ScreenPaths::ScreenPaths(std::string const& name, Audio& audio, Songs& songs): Screen(name), m_audio(audio), m_songs(songs) {}
 
 void ScreenPaths::enter() {
 	m_theme.reset(new ThemeAudioDevices());
-	m_txtinp.text.clear();
-
-	// FIXME: Temp error message
-	Game::getSingletonPtr()->dialog(
-		_("This tool is not yet available.\n"
-		  "Configure paths by adding them\n"
-		  "as command line parameters and\n"
-		  "then save them in configuration menu."));
+	fs::path homedir(getenv("HOME"));
```

Comment:
On windows vista / 7 it does that indeed, on windows XP it will be C:\Documents and settings\<username>\Documents